docs/framework/network-programming/tls.md

```diff
@@ -290,14 +290,14 @@ This table shows the OS update you'll need to support TLS 1.2 with .NET Framewor
 
 If you are using [Azure Cloud Services](https://azure.microsoft.com/services/cloud-services/) Web and Worker roles to host and run your application, there are some considerations that you need to take into account to support TLS 1.2.
 
-### .NET Framework 4.7 is not installed on Azure Guest OS by default
+### .NET Framework 4.7 is not installed on all Azure Guest OS families by default
 
-The latest version installed in the latest Azure Guest OS Family 5 release (Windows Server 2016) is 4.6.2. To see which versions of .NET Framework are installed on each Azure Guest OS, see the [Azure Guest OS releases and SDK compatibility matrix](https://docs.microsoft.com/azure/cloud-services/cloud-services-guestos-update-matrix).
+Azure Guest OS Family 6 release (Windows Server 2019) has .NET Framework 4.7.2 installed by default, but other Azure Guest OS family versions don't. To see which versions of .NET Framework are installed on each Azure Guest OS, see the [Azure Guest OS releases and SDK compatibility matrix](https://docs.microsoft.com/azure/cloud-services/cloud-services-guestos-update-matrix).
 
 If your app targets a .NET Framework version that is not available on the Azure Guest OS version, then you need to install it yourself. See [Install .NET on Azure Cloud Service Roles](https://docs.microsoft.com/azure/cloud-services/cloud-services-dotnet-install-dotnet). If the framework installation requires a restart, the service roles might also restart before entering the Ready state.
 
 ### Azure Guest OS registry settings
 
-The Azure Guest OS Family 5 image for [Azure Cloud Services](https://azure.microsoft.com/services/cloud-services/) already has the `SchUseStrongCrypto` registry key set to a value of 1. For more information, see [SchUseStrongCrypto](#schusestrongcrypto).
+The Azure Guest OS Family 6 image for [Azure Cloud Services](https://azure.microsoft.com/services/cloud-services/) already has the `SchUseStrongCrypto` registry key set to a value of 1. For more information about this registry key, see [SchUseStrongCrypto](#schusestrongcrypto).
```

Comment:
```suggestion
Starting with Azure Guest OS Family 5 for [Azure Cloud Services](https://azure.microsoft.com/services/cloud-services/) , the `SchUseStrongCrypto` registry key is already set to a value of 1. For more information about this registry key, see [SchUseStrongCrypto](#schusestrongcrypto).
```

Reply:
Do you know if this is accurate? The reason why 5 was mentioned was because it was the latest. So I just changed to 6.

Reply:
Family 5 has 4.6 installed, Family 6 has 4.7.1 so it would be enabled starting with 5. Family 2 is still officially supported and there are a lot of customers that will not (or can't) move to newer family versions.

However, I noted this in the article:

>If your app targets .NET Framework 4.6 or later versions, this key defaults to a value of 1. That's a secure default that we recommend. If your app runs on .NET Framework 4.6, but targets an earlier version, then the key defaults to 0. In that case, you should explicitly set its value to 1.

I think it will still be seen as 0 if your app targets 4.5 and runs on Family 5 (4.6), so the default is variable based on your target.

Reply:
@mairaw  We should close the loop on this PR. I still feel strongly that noting the defaults for the Azure Cloud Service offering is useful. We should just note the defaults for .NET and let the user figure out what version of .NET they are using on whatever service. There are Azure Functions, Azure VMs, Azure Containers, all of these may have different versions of .NET. There is no benefit from calling out Azure Cloud Services specifically.

Just for reference, here is the list of Cloud Service Guest OS: https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-guestos-update-matrix

These Cloud Services sections of the article may have made sense when Azure first started up and there was a single VM hosting service offered, and not much documentation, but that is no longer the case.

Reply:
So your suggestion still stands?

Reply:
I believe so. I think we should remove these sections from the article and not call out any specific azure service. There is no benefit to it.